### PR TITLE
[MM-28217] Front-end for displaying company info

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
System Console components to display Company Information from the CWS to the admins.
Edit is coming later.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28217

#### Related Pull Requests
- Has server changes (https://github.com/mattermost/mattermost-server/pull/15646)
- Has redux changes (https://github.com/mattermost/mattermost-redux/pull/1249)

### CODE REVIEW

**Comment by devinbinnie** [comment]
NOTE: Not ready for review until we have a way to setup a link to the CWS on the SpinWick.

**Review by nickmisasi — APPROVED** [review]
Very clean. LGTM!

File: components/admin_console/billing/company_info.tsx [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 type Props = {
 
 };
 
 const CompanyInfo: React.FC<Props> = () => {
+    const dispatch = useDispatch<DispatchFunc>();
+    const store = useStore();
+
+    useEffect(() => {
+        getCloudCustomer()(dispatch, store.getState());
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info.tsx:23]: Better to dispatch this action as opposed manually invoking the func.

```suggestion
        dispatch(getCloudCustomer());
```

File: components/admin_console/billing/company_info_display.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+.CompanyInfoDisplay__companyInfo-editButton {
+    color: rgba(var(--sys-center-channel-color-rgb), 0.56);
+    font-size: 18px;
+    
+    &:hover, &:focus {
+        text-decoration: none;
+        color: var(--sys-center-channel-color);
+    }
+}
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info_display.scss:135]: nit: empty line

**Review by ali-farooq0 — APPROVED** [review]

**Comment by marianunez** [comment]
> NOTE: Not ready for review until we have a way to setup a link to the CWS on the SpinWick.

Given that we still don't have the ability to set this up easily, let's merge and cherry pick this and we can do a `QA Review` and `UX Review` on the `cloud-ga` branch. I will be setting up a "epic PR" just so we can a do a full review

**Pull request merged**